### PR TITLE
Keyboard events sample in winui3

### PR DESCRIPTION
**Pull request by SreejaBhattacharya-MSFT**
## Description
This PR contains a sample that showcases the implementation of the below keyboard events in WinUI3 Windows App SDK sample app. It consists of both cs and cpp samples.
- KeyUp
- KeyDown
- CharacterReceived
- KeyboardAccelerators

## Checklist

- [x] Samples build and run using the Visual Studio versions listed in the [Windows development docs](https://docs.microsoft.com/windows/apps/windows-app-sdk/set-up-your-development-environment?tabs=stable#2-install-visual-studio).
- [x] Samples build and run on all supported platforms (x64, x86, ARM64) and configurations (Debug, Release).
- [x] Samples set the minimum supported OS version to Windows 10 version 1809.
- [x] Samples build clean with no warnings or errors.
- [x] **[For new samples]**: Samples have completed the [sample guidelines checklist](https://github.com/microsoft/WindowsAppSDK-Samples/blob/main/docs/samples-guidelines.md#checklist) and follow [standardization/naming guidelines](https://github.com/microsoft/WindowsAppSDK-Samples/blob/main/docs/samples-guidelines.md#standardization-and-naming).
- [x] I have commented on my PR `/azp run SamplesCI-<FeatureName>` to have the CI build run on my branch for each of my FeatureName my PR is modifying. This must be done on the latest commit on the PR before merging to ensure the build is up to date and accurate. Warning: the PR will not block automatically if this is not run due to '/azp run' limitation on triggering more than 10 pipelines.

### CODE REVIEW

**Comment by Saharsh979** [comment]
Windows App SDK samples are integrated with the [docs samples browser.](https://docs.microsoft.com/samples/browse/) All feature samples should include metadata in the sample README to integrate with the docs sample browser.

For reference: 
https://github.com/microsoft/WindowsAppSDK-Samples/tree/main/Samples/Widgets
https://github.com/microsoft/WindowsAppSDK-Samples/blob/main/Samples/Islands/README.md

**Comment by Saharsh979** [comment]
The guidelines recommend having samples for both C# and C++ at a minimum.

On a side note - There is not a hard requirement to cover all the other combinations (e.g. having all UI frameworks, having both Unpackaged and Packaged samples). However, if there is something unique about the specific component, the samples should highlight and showcase these scenarios.

**Comment by Saharsh979** [comment]
Add the appropriate copyright notice and license text to the headers.

For C#/C++ files (*h, *.cpp, *.cs, *.idl):

```
// Copyright (c) Microsoft Corporation.
// Licensed under the MIT License.
```
For XAML files (*.xaml):

```
<!-- Copyright (c) Microsoft Corporation.
 Licensed under the MIT License. -->
```

**Review by Saharsh979 — COMMENTED** [review]
Add the appropriate copyright notice and license text to the headers.

For C#/C++ files (*h, *.cpp, *.cs, *.idl):

```
// Copyright (c) Microsoft Corporation.
// Licensed under the MIT License.
```
For XAML files (*.xaml):

```
<!-- Copyright (c) Microsoft Corporation.
 Licensed under the MIT License. -->
```

File: Samples/Input/cs-winui/README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ extendedZipContent:
 
 # Input samples
 
-These samples demonstrate how to use the WinAppSDK input APIs within the namespace Microsoft.UI.Input.
+These samples demonstrate how to use the WinAppSDK input APIs within the namespace Microsoft.UI.Input. This sample also demonstrates the working of various Keyboard events and Keyboard accelerators using WinAppSDK.
```
[the remark this answers is not in the document]
Comment (SreejaBhattacharya-MSFT) [reply Samples/Input/cs-winui/README.md]: Done

**Comment by Saharsh979** [comment]
Has this step about "Onboarding Samples CI Pipeline for new feature" mentioned in the docs been taken care of?
I can see the .yml file is already present. _SamplesCI-Input.yml_
Kindly check if there are any pipeline changes required for this as well.

**Review by Saharsh979 — APPROVED** [review]

**Pull request merged**